Back-To-School/Assets/BackToSchool/Scripts/Constants.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Assets.BackToSchool.Scripts
+{
+    public static class Constants
```

Comment:
It seems a bit strange as for me - holding these enums in this class. There is no necessity for enums to be placed inside of class - just put them as a separate files in Enums folder